tests/WPUnit/OnboardingImportTest.php

```
@@ -0,0 +1,358 @@
+     *
+     * @dataProvider feedFailureProvider
+     */
+    public function testPreviewReportsFeedFailures($body, $status, $expected_code)
+    {
+        $this->feed_body   = $body;
+        $this->feed_status = $status;
+
+        $result = $this->handler->preview(self::FEED_URL);
+
+        $this->assertWPError($result);
+        $this->assertSame($expected_code, $result->get_error_code());
```

PHPStan (2.2.8)

[error] Cannot call method get_error_code() on array|WP_Error.


(method.nonObject)